src/stories/ScreenHeader.stories.tsx

```diff
@@ -0,0 +1,57 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import theme from '@styles/theme';
+import ScreenHeader from '@components/ScreenHeader';
+import SvgIcon from '@components/SvgIcon';
+
+const meta = {
+  title: 'Example/ScreenHeader',
+  component: ScreenHeader,
+  tags: ['autodocs'],
+} satisfies Meta<typeof ScreenHeader>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Resgister: Story = {
+  args: {
+    title: '스터디 생성',
+    leftIcon: <SvgIcon name="ArrowLeft" size={24} />,
+  },
+};
+
+export const NoTitle: Story = {
+  args: {
+    title: '',
+    leftIcon: <SvgIcon name="ArrowLeft" size={24} />,
+  },
+};
+
+export const DarkBg: Story = {
+  args: {
+    title: '어두운 배경',
+    titleColor: theme.palette.white,
+    leftIcon: (
+      <SvgIcon name="ArrowLeft" size={24} color={theme.palette.white} />
+    ),
+    rightIcon: (
+      <SvgIcon
+        name="ArrowLeft"
+        size={24}
+        color={theme.palette.white}
+        rotate={180}
+      />
+    ),
+  },
+  parameters: {
+    backgrounds: {
+      default: 'black',
+      values: [
+        {
+          name: 'black',
+          value: theme.palette.black,
```

Comment:
백그라운드 블랙은 따로 있다구~!!~!!
```suggestion
          value: theme.colors.background.black,
```

Reply:
아이공 변경해씀다 땡큐땡큐~!